### DISCUSSION TITLE
[Bug] Checkout shipment taxes label localization

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#425] [Help] [answered]
Ahoi. In the checkout process, the label is translated correctly but should be a capital letter "Versand". Where is this label coming from?

<img width="1092" alt="Screenshot 2023-11-02 at 16-58-34 Checkout" src="https://github.com/swellstores/community/assets/48205514/f5c4ff6e-4d94-4d91-8217-d193e1a467ee">

Additionally, if the shipment costs are 0, the label should not be there (or with a value 0.00).

<img width="1092" alt="Screenshot 2023-11-02 at 17-03-55 Checkout" src="https://github.com/swellstores/community/assets/48205514/3991458e-772f-49bd-b556-d0c915545956">

**Comment by logeshswell** [comment]
~Can you share the cart link please?~

Please ignore it as I found the cart link.

**Reply by logeshswell** [reply]
> Additionally, if the shipment costs are 0, the label should not be there (or with a value 0.00).

The current/expected behavior represents nil charges using a hyphen. But, I will raise an improvement request to represent it as **CHF 0.00** for easy understanding.

**Reply by logeshswell** [reply, marked as answer]
> but should be a capital letter "Versand"

It is labeled as lower case letter for a purpose and it is possible to overwrite it using checkout scripts.

> Where is this label coming from?

The value is fetched from the Checkout editor under Checkout settings > Shopping cart labels.
